### PR TITLE
feat(vps.upscale): enable the configuration of all upscale flavors.

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yno
| Tickets          | Fix #MANAGER-7321 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/vps/src/upscale/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -30,9 +32,9 @@
   "vps_upscale_summary_final_configuration_upfront": "Configuration finale pour un engagement de {{ month }} mois",
   "vps_upscale_summary_price_ex_vat_per_month": "Prix HT/mois",
   "vps_upscale_summary_price_ex_vat": "Prix HT",
-  "vps_upscale_summary_price_monthly": "Prix au prorata pour le mois en cours",
+  "vps_upscale_summary_price_monthly": "Prix au prorata temporis jusqu'à la fin de votre abonnement actuel",
   "vps_upscale_summary_price_upfront": "Prix pour temps d‘engagement restant",
-  "vps_upscale_summary_price_monthly_with_payment_method_validation": "En validant ce choix, vous serez facturé(e) de {{ price }} au prorata temporis pour le mois en cours.",
+  "vps_upscale_summary_price_monthly_with_payment_method_validation": "A la fin de votre abonnement, vous pourrez renouveler votre service au tarif de {{ price }} /mois HT.",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/vps/src/upscale/translations/Messages_fr_FR.json:37]:
```suggestion
  "vps_upscale_summary_price_monthly_with_payment_method_validation": "À la fin de votre abonnement, vous pourrez renouveler votre service au tarif de {{ price }} /mois HT.",
```

File: packages/manager/modules/vps/src/upscale/upscale.controller.js [hunk 2]
```diff
@@ -180,9 +176,17 @@ export default class UpscaleController {
     );
   }
 
-  getRangeEliteConfigurationPricing() {
+  isStorageSelectionDisabled(storage) {
+    const { cores, memory } = this.rangeConfiguration;
+    const rangeName = this.range.formattedName.toLowerCase();
+    const plans = this.catalog.plans.map((plan) => plan.planCode);
+    this.planCode = `vps-${rangeName}-${cores}-${memory}-${storage}`;
```
Comment (mohammed-zahaf) [inline packages/manager/modules/vps/src/upscale/upscale.controller.js:183]: regarding name of function `isStorageSelectionDisabled` we expect boolean as return.
but in same function you are assigned a value to `this.planCode`.

I guess you have to assign the value somewhere else.

File: packages/manager/modules/vps/src/upscale/translations/Messages_fr_FR.json (hunk 1 above)
Comment (brofik) [reply packages/manager/modules/vps/src/upscale/translations/Messages_fr_FR.json]: Done.

**Review by mohammed-zahaf — DISMISSED** [review]

File: packages/manager/modules/vps/src/upscale/upscale.controller.js (hunk 2 above)
Comment (anooparveti) [reply packages/manager/modules/vps/src/upscale/upscale.controller.js:183]: @brofik, @mohammed-zahaf's comment is valid. Can you please look into it?

**Pull request closed without merging**